index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Primary Meta Tags -->
+    <title>Linkbrary</title>
+    <meta name="title" content="Linkbrary" />
+    <meta name="description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://linkbrary-seo.netlify.app/" />
+    <meta property="og:title" content="Linkbrary" />
+    <meta property="og:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta property="og:image" content="https://linkbrary-seo.netlify.app/images/preview.png" />
+    <!-- Twitter -->
+    <meta property="twitter:card" content="photo" />
+    <meta property="twitter:url" content="https://linkbrary-seo.netlify.app/" />
+    <meta property="twitter:title" content="Linkbrary" />
+    <meta property="twitter:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta property="twitter:image" content="https://linkbrary-seo.netlify.app/images/preview.png" />
+    <link rel="stylesheet" href="css/Linkbrary_style.css">
+    <link href="https://webfontworld.github.io/pretendard/Pretendard.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Abel&family=Acme&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <nav>
+      <div class="navbar">
+        <a href="index.html"><img src="images/logo.svg"></a>
+        <a class="sign-in" href="signin.html"><span>로그인</span></a>
+      </div>
+    </nav>
+
+    <main>
+      <header>
+        <div class="headline">
+          <h1>
+            <span class="headline-title accent">세상의 모든 정보</span>를<br>
```

Comment:
좀 더 대상을 나타내는 클래스명이면 좋을 것 같아요. 아래는 예시이니 한번 고민해보시면 좋겠습니다~

```suggestion
            <span class="headline-keyword accent">세상의 모든 정보</span>를<br>
```